docs/best-practices/continuous-integration/bitrise.md

```diff
@@ -0,0 +1,60 @@
+# Bitrise Integration
+
+[Bitrise](https://www.bitrise.io) allows you to automatically build, test and deploy your iOS and Android apps.
+
+## How to get started?
+
+Using _fastlane_ for your Bitrise workflow is simple. Just [add the ](https://devcenter.bitrise.io/getting-started/manage-your-bitrise-workflow) [_fastlane_](https://www.bitrise.io/integrations/steps/fastlane) [step to your
```

Comment:
The linking going on here is pretty confusing. Can this be rewritten so these two links have a word in between somehow and a link is not repeated?

(also: additional space after `add the` that should be removed)